src/main/flight/pid.h

```diff
@@ -85,11 +86,6 @@ typedef struct pidProfile_s {
     uint16_t rateAccelLimit;                // accel limiter roll/pitch deg/sec/ms
 } pidProfile_t;
 
-#if FLASH_SIZE <= 128
```

Comment:
not how i understood it, the code simply was moved from pid.h into common_fc_pre.h.  The latter file is where most per-flash-size settings are made so moving it made lots of sense to me.  Additionally MAX_PROFILE_COUNT should not be used for the OSD_SLAVE targets since they do not use profiles.